package.json

```diff
@@ -27,7 +27,8 @@
     "react-dom": "^18.2.0",
     "react-icons": "^4.4.0",
     "styled-components": "^5.3.5",
-    "tailwindcss": "^3.1.6"
+    "tailwindcss": "^3.1.6",
+    "yarn": "^1.22.19"
```

Comment:
You are installed YARN with a project dependency, but it is a global dependency for u. 

npm install -g yarn

and remove this line.